dipy/workflows/reconst.py

```diff
@@ -21,6 +21,114 @@
 from dipy.reconst.shm import CsaOdfModel
 from dipy.workflows.workflow import Workflow
 
+from dipy.reconst import mapmri
+import matplotlib
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+class ReconstMAPMRIFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'mapmri'
+
+    def run(self, data_file, data_bvecs, data_bvals, small_delta=0.0129, big_delta=0.0218
+            save_metrics = [], out_dir='', out_mapmri='MAPMRI_maps_regularization.png'):  
+        """ Workflow for the app-dipy-mapmri on Brain-Life (www.brain-life.org). 
+        Generates RTOP graphs saved in a .png format in input files provided by 
+        `data_file` and saves the png file to an output directory specified by 
+        `out_dir`.
+
+        Parameters
+        ----------
+        data_file : string
+            Path to the input volume.
+        data_bvecs : string
+            Path to the bvec files.
+        data_bvals : 
+            Path to the bval files.
+        small_delta :
+            Small delta value used in generation of gradient table of provided
+            bval and bvec. (default: 0.0129)
+        big_delta :
+            Big delta value used in generation of gradient table of provided 
+            bval and bvec. (default: 0.0218)
+        save_metrics :
+            List of metrics to save.
+            Possible values: mmri
+            (default: [] (all))
+        out_dir : string, optional
+            Output directory (default: input file directory)
+        out_mapmri : string, optional
+            Name of the png file to be saved (default: MAPMRI_maps_regularization.png))
+        """
+        io_it = self.get_io_iterator()
+        for dwi, bval, bvec, mapname in io_it:
+
+            logging.info('Computing DTI metrics for {0}'.format(dwi))
+            img = nib.load(dwi)
+            data = img.get_data()
+            bvals,bvecs = read_bvals_bvecs(bval, bvec)
+
+        
+            gtab = gradient_table(bvals=bvals, bvecs=bvecs,
+                                small_delta=small_delta,
+                                big_delta=big_delta, b0_threshold=50)
+
+
+            data_small = data[60:85, 80:81, 60:85]
```

Comment:
Why do you crop the data? if it is really necessary, you have to warn the user that only a small part of its data will be processed. Otherwise, use the whole data.